modules/aws/aurora-serverless/variables.tf

```diff
@@ -114,11 +114,13 @@ variable "create_db_cluster_parameter_group" {
 variable "db_cluster_parameter_group_family" {
   description = "The family of the DB cluster parameter group"
   type        = string
-  default     = "aurora-mysql5.7"
-  validation {
-    condition     = var.db_cluster_parameter_group_family == "aurora-mysql5.7" || var.db_cluster_parameter_group_family == "aurora-postgresql10"
-    error_message = "The value choosen is not in the list of ( aurora-mysql5.7, aurora-postgresql10)."
-  }
+  default     = "aurora-postgresql13"
+}
+
+variable "engine_version" {
+  description = "The database engine version. Updating this argument results in an outage"
+  type        = string
+  default     = "13.9"
```

Comment:
This would be only valid when both `db_cluster_parameter_group_family` and `engine_version` are empty, because if the DB family is set to MySQL, and an engine version is not specified correctly, then the module will try to provision a MySQL instance with DB version to `13.9`, which is invalid.

This would be handled at the frontend (UI) side, but is there anything we can do to improve this, or should we go with the current behavior of using `13.9` as the default one?